### ISSUE TITLE
Playlist API simplification

### ISSUE THREAD

**Issue by Zeugma440** [Zeugma440/atldotnet#233]
It'd also be nice if you could add a `Save` function to the PlaylistIO API.

Currently if we want to add or remove paths from a playlist we have to do:
```csharp
var paths = _playlist.FilePaths;
paths.Remove(0);
paths.Add("Test.mp3");
paths.Add("Test2.mp3");
_playlist.FilePaths = paths;
```
since it is the setter of `FilePaths` that causes the changes to be written to the actually playlist file on disk.

It'd be nice to have a more friendly API such as the following:
```csharp
_playlist.FilePaths.Remove(0);
_playlist.FilePaths.Add("Test.mp3");
_playlist.FilePaths.Add("Test2.mp3");
_playlist.Save();
```
I mean currently, we can do this:
```csharp
_playlist.FilePaths.Remove(0);
_playlist.FilePaths.Add("Test.mp3");
_playlist.FilePaths.Add("Test2.mp3");
_playlist.FilePaths = _playlist.FilePaths;
```
but, as I said before, since the FilePaths setter is what writes the changes, it feels cumbersome doing it this way.

**Comment by Zeugma440** [comment]
Not sure I wanna do this, as it would require my `Playlist` class to store and compare values, which introduces too much complexity for the sake of a slightly better API. The tradeoff looks bad to me.

Would you accept to settle for something like :
```cs
var paths = _playlist.FilePaths;
paths.Remove(0);
paths.Add("Test.mp3");
paths.Add("Test2.mp3");
_playlist.Save(paths);
```

**Comment by nlogozzo** [comment]
that works :)

**Comment by Zeugma440** [comment]
Available on today's v5.09
